compiler/rustc_middle/src/ty/sty.rs

```diff
@@ -1436,7 +1436,7 @@ pub struct ConstVid<'tcx> {
 rustc_index::newtype_index! {
     /// A **region** (lifetime) **v**ariable **ID**.
     #[derive(HashStable)]
-    #[debug_format = "'_#{}r"]
+    #[debug_format = "'?{}"]
```

Comment:
Why regions in particular are unsuffixed? All other vids seem to have suffices denoting their type (c = const, t = type, i = int, f = float)

Reply:
nvm, that's because they are prefixed with `'`

Reply:
It could go either way, preferrred to have shorter region vars.